test/verify/check-connection

```diff
@@ -450,6 +450,13 @@ class TestConnection(MachineCase):
         output = m.execute('curl -s -S -k https://172.27.0.15:9000/ 2>&1 || true')
         self.assertIn('Connection refused', output)
 
+        # Large requests are processed correctly with plain HTTP
+        self.assertIn('A Custom Title', m.execute('''curl -s -S -H "Authorization: Negotiate $(printf '%0.7000i' 1)" http://localhost:9000/'''))
+
+        # Large requests are processed correctly with TLS
```

Comment:
In effect, this will be testing cockpit-tls, correct?

Reply:
No, it tests cockpit-ws directly; the code above runs `cockpit_ws_executeable`. That's why it fails on 8-1-distropkg. We could add a test case through cockpit-tls as well of course, I can do that in a follow-up. I have another PR in the pipeline for adding a new unit test anyway.